🐛 [`WatchOptions`] `Cannot find module '' or its corresponding type declarations`

### Prerequisites

- [X] read documentation
- [X] searched open issues for existing bug report
- [X] searched closed issues for solution(s) or feedback

### Current behavior

Type checking a file that uses [`WatchOptions`][1] produces an error:

```zsh
node_modules/@flex-development/tsconfig-types/dist/types/watch-options.d.mts:6:62 - error TS2307: Cannot find module '' or its corresponding type declarations.

6 import type { PollingWatch, WatchDirectory, WatchFile } from '';
                                                               ~~

Found 1 error in node_modules/@flex-development/tsconfig-types/dist/types/watch-options.d.mts:6
```


### Reproduction

1. `git clone https://github.com/flex-development/mlly`
2. `cd mlly`
3. `nvm use`
4. `yarn --immutable`
5. `yarn add @flex-development/tsconfig-types @flex-development/pathe`
6. Update `src/internal/get-compiler-options.ts`

    ```typescript
    /**
     * @file Internals - getCompilerOptions
     * @module mlly/internal/getCompilerOptions
     */
    
    import pathe from '@flex-development/pathe'
    import type { CompilerOptions } from '@flex-development/tsconfig-types'
    import { loadTsconfig } from 'tsconfig-paths/lib/tsconfig-loader'
    
    /**
     * Retrieves TypeScript compiler options from `path`.
     *
     * Supports [`extends`][1].
     *
     * [1]: https://typescriptlang.org/tsconfig#extends
     *
     * @internal
     *
     * @param {string} [path=pathe.resolve('tsconfig.json')] - Tsconfig path
     * @param {(path: string) => boolean} [exists] - File existence checker
     * @param {(filename: string) => string} [read] - File content reader
     * @return {CompilerOptions} User compiler options
     */
    const getCompilerOptions = (
      path: string = pathe.resolve('tsconfig.json'),
      exists?: (path: string) => boolean,
      read?: (filename: string) => string
    ): CompilerOptions => {
      /**
       * Tsconfig object.
       *
       * @const {{ compilerOptions?: CompilerOptions } | undefined} t
       */
      const t: { compilerOptions?: CompilerOptions } | undefined = loadTsconfig(
        path,
        exists,
        read
      )
    
      return !t?.compilerOptions ? {} : t.compilerOptions
    }
    
    export default getCompilerOptions
    ```
7. `vue-tsc ./src/internal/get-compiler-options.ts --noEmit`
8. See error


### Expected behavior

Type checking should not produce any errors.

### Package version

`2.0.0`

### Node.js version

`v16.17.0`

### Package manager version

`4.0.0-rc.14`

### Operating system (environment)

macos

### Additional context

N/A

[1]: https://github.com/flex-development/tsconfig-types/blob/2.0.0/src/types/watch-options.ts

--- a/src/types/__tests__/watch-options.spec-d.ts
+++ b/src/types/__tests__/watch-options.spec-d.ts
@@ -8,8 +8,10 @@ import type {
   WatchDirectoryKind,
   WatchFileKind
 } from '#src/enums'
-import type { PollingWatch, WatchDirectory, WatchFile } from '#src/types'
 import type { JsonObject, KeysRequired } from '@flex-development/tutils'
+import type PollingWatch from '../polling-watch'
+import type WatchDirectory from '../watch-directory'
+import type WatchFile from '../watch-file'
 import type TestSubject from '../watch-options'
 
 describe('unit:interfaces/WatchOptions', () => {

--- a/src/types/watch-options.ts
+++ b/src/types/watch-options.ts
@@ -8,7 +8,9 @@ import type {
   WatchDirectoryKind,
   WatchFileKind
 } from '#src/enums'
-import type { PollingWatch, WatchDirectory, WatchFile } from '#src/types'
+import type PollingWatch from './polling-watch'
+import type WatchDirectory from './watch-directory'
+import type WatchFile from './watch-file'
 
 /**
  * File and directory watching options.

